playwright/_impl/_clock.py

```diff
@@ -25,22 +25,22 @@ def __init__(self, browser_context: "BrowserContext") -> None:
         self._loop = browser_context._loop
         self._dispatcher_fiber = browser_context._dispatcher_fiber
 
-    async def install(self, time: Union[int, str, datetime.datetime] = None) -> None:
+    async def install(self, time: Union[float, str, datetime.datetime] = None) -> None:
```

Comment:
Why is it float? Figuring out if this will confuse Python folks that use seconds for time.